openapi/openapi.yaml

```diff
@@ -697,6 +697,50 @@ components:
       pattern: ^((([0-9]+)\.([0-9]+)\.([0-9]+)(?:-([0-9a-zA-Z-]+(?:\.[0-9a-zA-Z-]+)*))?)(?:\+([0-9a-zA-Z-]+(?:\.[0-9a-zA-Z-]+)*))?)$
       example: '1.0.1'
 
+    Tag:
+      description: 'Tag'
+      title: 'Tag'
+      type: object
+      properties:
+        name:
+          maxLength: 32
+          type: string
+        id:
+          description: 'Database ID for this object.'
+          readOnly: true
+          type: integer
+        url:
+          description: 'URL for this resource.'
+          readOnly: true
+          type: string
+          format: uri
+        modified:
```

Comment:
Tags are supposed to be read only. What does `modified` time mean for tag exactly?